test/SparqlAntlrParserTest.cpp

```diff
@@ -598,6 +598,29 @@ TEST(SparqlParser, VarOrTermGraphTerm) {
   EXPECT_THAT(varOrTerm, IsIri(nil));
 }
 
+TEST(SparqlParser, Iri) {
+  auto expectIri = [](const string& input, string iri,
+                      SparqlQleverVisitor::PrefixMap prefixMap = {}) {
+    // Parse "by hand" in order not to pollute the `SparqlParserHelpers`.
+    ParserAndVisitor p{input, std::move(prefixMap)};
+    expectCompleteParse(
+        p.parseTypesafe(input, "iri", &SparqlAutomaticParser::iri),
+        testing::Eq(std::move(iri)));
```

Comment:
`std::move` is redundant with `const &` (it doesn't actually move). Doesn't clang-tidy complain about this?

Reply:
`iri` is not `const &`. Will fix that.

Reply:
Ok, that is correct.
But do the Matchers consume their arguments, or should `iri` be a const&
(not that it really matters in a unit test, but in C++ in general we have to be aware of these things.

Reply:
I think `const&` should be the type here aswell.